ppb/systems/__init__.py

```diff
@@ -99,8 +101,9 @@ def prepare_rectangle(self, resource, game_object, camera):
 
     def register(self, resource_path, name=None):
```

Comment:
I feel like we probably want to rename the parameter here? Since it's not resource_path anymore?

Reply:
Oops! This code actually got totally nixed. I forgot to mark this PR as WIP.